src/components/ShareStructure/ShareStructure.vue

```diff
@@ -434,30 +434,30 @@ export default class ShareStructure extends Mixins(ShareMixin) {
   // Props
   /** Edit Mode */
   @Prop({ default: true })
-  private isEditMode!: boolean
+  readonly isEditMode!: boolean
 
-  @Prop()
-  private isCorrection: boolean
+  @Prop({ default: false })
+  readonly isCorrection: boolean
 
   @Prop({ default: null })
-  private incorporationApplication: IncorporationFilingIF
+  readonly incorporationApplication: IncorporationFilingIF
 
   @Prop({ default: null })
-  private businessSnapshot: BusinessSnapshotIF[]
+  readonly originalShareStructure: ShareStructureIF
```

Comment:
Passing in just the Share Structure, so that THIS component doesn't need to know about the business snapshot.